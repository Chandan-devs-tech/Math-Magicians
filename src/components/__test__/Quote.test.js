import React from 'react';
import renderer from 'react-test-renderer';
import Quotes from '../Quote';

describe('Quotes', () => {
  test('should render the component', () => {
    const component = renderer.create(<Quotes />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
