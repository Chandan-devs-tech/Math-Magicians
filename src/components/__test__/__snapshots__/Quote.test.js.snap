// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Calculator should render the component 1`] = `
<div
  className="calcText"
>
  <h2>
    Let's do some math!
  </h2>
  <div
    className="calc"
  >
    <div
      className="input"
    >
      0
    </div>
    <div
      className="row"
    >
      <button
        className="btn "
        onClick={[Function]}
        type="button"
      >
        AC
      </button>
      <button
        className="btn "
        onClick={[Function]}
        type="button"
      >
        +/-
      </button>
      <button
        className="btn "
        onClick={[Function]}
        type="button"
      >
        %
      </button>
      <button
        className="btn orangeColor"
        onClick={[Function]}
        type="button"
      >
        ÷
      </button>
    </div>
    <div
      className="row"
    >
      <button
        className="btn "
        onClick={[Function]}
        type="button"
      >
        7
      </button>
      <button
        className="btn "
        onClick={[Function]}
        type="button"
      >
        8
      </button>
      <button
        className="btn "
        onClick={[Function]}
        type="button"
      >
        9
      </button>
      <button
        className="btn orangeColor"
        onClick={[Function]}
        type="button"
      >
        x
      </button>
    </div>
    <div
      className="row"
    >
      <button
        className="btn "
        onClick={[Function]}
        type="button"
      >
        4
      </button>
      <button
        className="btn "
        onClick={[Function]}
        type="button"
      >
        5
      </button>
      <button
        className="btn "
        onClick={[Function]}
        type="button"
      >
        6
      </button>
      <button
        className="btn orangeColor"
        onClick={[Function]}
        type="button"
      >
        -
      </button>
    </div>
    <div
      className="row"
    >
      <button
        className="btn "
        onClick={[Function]}
        type="button"
      >
        1
      </button>
      <button
        className="btn "
        onClick={[Function]}
        type="button"
      >
        2
      </button>
      <button
        className="btn "
        onClick={[Function]}
        type="button"
      >
        3
      </button>
      <button
        className="btn orangeColor"
        onClick={[Function]}
        type="button"
      >
        +
      </button>
    </div>
    <div
      className="row"
    >
      <button
        className="zeroBtn btn "
        onClick={[Function]}
        type="button"
      >
        0
      </button>
      <button
        className="btn "
        onClick={[Function]}
        type="button"
      >
        .
      </button>
      <button
        className="btn orangeColor"
        onClick={[Function]}
        type="button"
      >
        =
      </button>
    </div>
  </div>
</div>
`;

exports[`Quotes should render the component 1`] = `
Array [
  <div
    className="double-quotes"
  >
    “
  </div>,
  <div
    className="quoteContainer"
  >
    <p />
  </div>,
]
`;
